main.py

```diff
@@ -0,0 +1 @@
+print('I'm the main file')
```

Comment:
Тут є помилка через використання одинарних лапок в середені рядка.
Я пропоную цю строку зробити такою:
```suggestion
print("I'm the main file")
```